src/main/java/net/openhft/chronicle/threads/AbstractLifecycleEventLoop.java

```diff
@@ -43,12 +43,21 @@ public abstract class AbstractLifecycleEventLoop extends AbstractCloseable imple
      * tests don't block forever. This time should be kept as "effectively forever".
      */
     private static final long AWAIT_TERMINATION_TIMEOUT_MS = TimeUnit.MINUTES.toMillis(5);
-    private final AtomicReference<EventLoopLifecycle> lifecycle = new AtomicReference<>(EventLoopLifecycle.NEW);
     protected final String name;
+    private final AtomicReference<EventLoopLifecycle> lifecycle = new AtomicReference<>(EventLoopLifecycle.NEW);
+
+    private final Pauser pauser;
```

Comment:
A better approach IMO would be to allow event loops to describe themselves (e.g. add EventLoop#shortDescription()) or something, that way we don't have to expose internal EventLoop infrastructure to the outside world to get these details out. Also it would allow specific event loops to have a specific short description tailored to it's own internals.

Reply:
And we already have `EventLoopStateRenderer` too, although I don't know how relevant that is